### DISCUSSION TITLE
Please support Ottai sensor.

### DISCUSSION THREAD

**Discussion by SimonXuku** [JohanDegraeve/xdripswift#546] [General]
Ottai only sells for $18-20,it's cheap and accurate, a little slower than G6. 

Hope to add this sensor to loop via xdripswift.

Blow is the official website, it provides android and ios version.

https://ottai.com/en/

**Comment by paulplant** [comment]
Sorry but no idea about this sensor - it's not known in the DIY diabetes community and no communication protocol/API is known/released.
